nnunetv2/training/nnUNetTrainer/nnUNetTrainer.py

```diff
@@ -619,12 +619,13 @@ def get_dataloaders(self):
             mt_gen_val = SingleThreadedAugmenter(dl_val, val_transforms)
         else:
             mt_gen_train = LimitedLenWrapper(self.num_iterations_per_epoch, data_loader=dl_tr, transform=tr_transforms,
-                                             num_processes=allowed_num_processes, num_cached=6, seeds=None,
+                                             num_processes=allowed_num_processes, num_cached=100, seeds=None,
```

Comment:
I wonder if we should pull it out into a CLI or env var?

Reply:
I think so too yes.